scripts/set-manifest-content-script.js

```diff
@@ -5,9 +5,15 @@ const manifestPath = "./build/manifest.json"
 const run = async () => {
   const json = await fs.promises.readFile(manifestPath, "utf-8")
   const manifest = JSON.parse(json)
+  
+  // Ensure URL has scheme
+  const baseUrl = process.env.REACT_APP_SHARE_TARGET_URL || 'http://localhost:3000'
+  const url = baseUrl.startsWith('http') ? baseUrl : `http://${baseUrl}`
```

Comment:
Might be better to default to https with these checks. As the REACT_APP_SHARE_TARGET_URL will always be a secure url.